hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/ConsensusServiceSimpleFeesSuite.java

```diff
@@ -0,0 +1,305 @@
+                                    .fee(ONE_HBAR)
+                                    .via("update-topic-txn")),
+                    "update-topic-txn",
+                    0.000310,
+                    1,
+                    1000);
+        }
+
+        @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+        @DisplayName("compare update topic with admin key")
+        final Stream<DynamicTest> updateTopicComparisonWithAdmin() {
+            final String ADMIN = "admin";
+            return compareSimpleToOld(
+                    () -> Arrays.asList(
+                            newKeyNamed(ADMIN),
+                            cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                            createTopic("testTopic")
+                                    .blankMemo()
+                                    .payingWith(PAYER)
+                                    .adminKeyName(ADMIN)
+                                    .fee(ONE_HBAR)
+                                    .via("create-topic-admin-txn"),
+                            updateTopic("testTopic")
+                                    .adminKey(ADMIN)
+                                    .payingWith(PAYER)
+                                    .fee(ONE_HBAR)
+                                    .via("update-topic-txn")),
+                    "update-topic-txn",
+                    0.00131,
+                    1,
+                    100);
+        }
+
+        @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+        @DisplayName("compare submit message with included bytes")
+        final Stream<DynamicTest> submitMessageFeeWithIncludedBytesComparison() {
+            // 100 is less than the free size, so there's no per byte charge
+            final var byte_size = 100;
+            final byte[] messageBytes = new byte[byte_size]; // up to 1k
+            Arrays.fill(messageBytes, (byte) 0b1);
+            return compareSimpleToOld(
+                    () -> Arrays.asList(
+                            newKeyNamed(PAYER),
+                            cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                            // create topic, provide up to 1 hbar to pay for it
+                            createTopic("testTopic")
+                                    .blankMemo()
+                                    .payingWith(PAYER)
+                                    .fee(ONE_HBAR)
+                                    .via("create-topic-txn"),
+                            // submit message, provide up to 1 hbar to pay for it
+                            submitMessageTo("testTopic")
+                                    .blankMemo()
+                                    .payingWith(PAYER)
+                                    .message(new String(messageBytes))
+                                    .fee(ONE_HBAR)
+                                    .via("submit-message-txn")),
+                    "submit-message-txn",
+                    0.0001900,
```

Comment:
Shouldn't this be exactly base price? Since 100 bytes is allowed and we only have one signature